app/controllers/feeds_controller.rb

```diff
@@ -4,9 +4,7 @@ def show
     @repository = RepositorySerializer.perform(data)
     SubscriptionLogger.perform(request)
 
-    respond_to do |format|
```

Comment:
It honestly didn't seem to do anything. I removed it and everything worked as normal. I suspect it's because there's only one format, and I [default the format](https://github.com/edwardloveall/pullfeed/blob/master/config%2Froutes.rb#L6) in the routes.